lib/src/executable.dart

```diff
@@ -94,25 +104,142 @@ Future<void> handleFastFormat(List<String> args) async {
   }
 }
 
-void generateRunScript() {
-  if (shouldWriteRunScript) {
+void _deleteRunExecutableAndDigest() =>
+    [_paths.runExecutable, _paths.runExecutableDigest].forEach((p) {
+      final f = File(p);
+      if (f.existsSync()) f.deleteSync();
+    });
```

Comment:
Security review: The files to delete look like they are statically defined, so this couldn't be used to delete something unexpected:  https://github.com/Workiva/dart_dev/pull/426/files#diff-6e5f65bcac0e9f8da735ef58d443983df38419788614afe4a860adebb5016f5cR35-R37